mne/io/eeglab/eeglab.py

```diff
@@ -119,6 +117,30 @@ def read_raw_eeglab(input_fname, montage=None, preload=False, eog=(),
         Path or instance of montage containing electrode positions.
         If None, sensor locations are (0,0,0). See the documentation of
         :func:`mne.channels.read_montage` for more information.
+    eog : list | tuple | 'auto'
+        Names or indices of channels that should be designated EOG channels.
+        If 'auto', the channel names containing ``EOG`` or ``EYE`` are used.
+        Defaults to empty tuple.
+    misc : list or tuple
+        Names of channels or list of indices that should be designated MISC
+        channels. Values should correspond to the electrodes in the vhdr file.
+        Default is ``()``.
+    event_id : dict | None
+        The ids of the events to consider. If None (default), an empty dict is
+        used and `event_id_func` (see below) is called on every event value.
+        If dict, the keys will be mapped to trigger values on the stimulus
+        channel and only keys not in `event_id` will be handled by
+        `event_id_func`. Keys are case-sensitive.
+        Example::
+
+            {'SyncStatus': 1; 'Pulse Artifact': 3}
+
+    event_id_func : None | callable
+        What to do for events not found in `event_id`. Must take one `str`
+        argument and return an `int`. If `None`, defaults to stripping event
+        codes such as "D128" or "S  1" of their non-integer parts and returns
+        the integer. Any event that is not in `event_id` and cannot be parsed
+        with this function is dropped.
```

Comment:
Do we actually need two separate functions for `event_id` and `event_id_func`. I don't want to repeat any of the old discussion. Maybe @jona-sassenhagen can you remind me why we chose to have two different arguments instead of `event_id : dict | callable | None`?